tests/jballerina-integration-test/src/test/resources/websocket/src/wsservices/37_cookie_client.bal

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+
+http:ClientConfiguration clientEPConfig = {
+    cookieConfig: {
+        enabled: true
+    }
+};
+
+@http:WebSocketServiceConfig {
+}
+service on new http:Listener(21037) {
+
+    resource function onOpen(http:WebSocketCaller wsEp) {
+        http:Client httpClient = new("http://localhost:21036/cookie-demo", clientEPConfig);
+
+        http:Request request = new;
+
+        json jsonPart = {
+            name: "John",
+            password: "p@ssw0rd"
+        };
+        request.setJsonPayload(jsonPart);
+
+        var loginResp = httpClient->post("/login", request);
+
+        if (loginResp is http:Response) {
+            string|error loginMessage = loginResp.getTextPayload();
```

Comment:
Why do we need to access payload here as we use `getCookies()` API to access cookies?

Reply:
If the server accepts the login request, it will set the string [payload](https://github.com/ballerina-platform/ballerina-distribution/pull/668/files#diff-40b6aad65e7d45ea424e4fa2e5e8358bR43) in the response. So here, we check whether getting that string in the payload or not